### PR TITLE
New serverless pattern - Serverless text and image Generation with Amazon Bedrock using SAM

### PR DESCRIPTION
**Pull request by sushir-vr**
*Issue #1696, if available:*

*Description of changes:*
This pattern creates an API Gateway REST API with Lambda proxy integration which places invoke calls to Amazon Bedrock for text and image generation. Bedrock allows the use of various LLMs by providers like AI21 labs, Anthropic etc which can be used to generate texts and images in repsonse to prompts provided by users.
The API generated in this pattern exposes multiple text and image generation models currently available in Bedrock to allow users to send requests to any of the available models.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sushir-vr** [comment]
Fixed validity check errors

**Comment by boyney123** [comment]
Hey,

Looks like this branch has merge conflicts? Any chance you can take a look at them?

File: apigw-lambda-bedrock-sam/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    "text": [
+      "Delete the stack: <code>sam delete</code>."
+    ]
+  },
+  "authors": [
+    {
+      "name": "Abha Tripathi",
+      "image": "https://avatars.githubusercontent.com/u/36122588?s=400&u=209552eb3fa55e5c4176e54a24a7737ce899083b&v=4",
+      "bio": "Cloud Engineer @ AWS",
+      "linkedin": "www.linkedin.com/in/abha-tripathi-344a0a146"
```
Comment (boyney123) [inline apigw-lambda-bedrock-sam/example-pattern.json:52]:
```suggestion
      "linkedin": "abha-tripathi-344a0a146"
```

File: apigw-lambda-bedrock-sam/example-pattern.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    {
+      "name": "Sushir V R",
+      "image": "https://avatars.githubusercontent.com/u/42473846?s=400&u=4c7b19858e4ec3d9ae24e379c35848219ee57a86&v=4",
+      "bio": "Cloud Engineer @ AWS",
+      "linkedin": "www.linkedin.com/in/sushirvr"
```
Comment (boyney123) [inline apigw-lambda-bedrock-sam/example-pattern.json:58]:
```suggestion
      "linkedin": "sushirvr"
```

**Review by boyney123 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**